docs/guide/api-environment.md

```diff
@@ -1,7 +1,7 @@
 # Environment API
 
 :::warning Experimental
-Initial work for this API was introduced in Vite 5.1 with the name "Vite Runtime API". This guide describes a revised API, renamed to Environment API. This API will be released in Vite 6 as experimental. You can already test it in the latest `vite@6.0.0-beta.x` version.
+Initial work for this API was introduced in Vite 5.1 with the name "Vite Runtime API". This guide describes a revised API, renamed to Environment API. This API was released as an experimental release in `Vite 6`. You can already test it in the latest `vite@6.0.0-beta.x` version.
```

Comment:
I think we shouldn't reference Vite Runtime API anymore. It is already in the migration guide, and for most users, Environment API will be the first thing they use.


```suggestion
Environment API is experimental. We'll keep the APIs stable during Vite 6 to let the ecosystem experiment and build on top of it. We're planning to stabilize these new APIs with potential breaking changes in Vite 7.
```

At the end of warning, it should be changed from "Please share with us your feedback as you test the proposal." to "Please share with us your feedback."

We also need to do the same changes to every Environment API Guide (there are 5 of them)

Reply:
Okay, I did it

Reply:
Thanks! Could you also change this part: 
> At the end of warning, it should be changed from "Please share with us your feedback as you test the proposal." to "Please share with us your feedback."

Reply:
ok i did it !